docs/howto/openstack/octavia/metrics.md

```diff
@@ -0,0 +1,151 @@
+    --protocol-port 8088 \
+    --allowed-cidrs "203.0.113.132/32" \
+    mylb
++-----------------------------+--------------------------------------+
+| Field                       | Value                                |
++-----------------------------+--------------------------------------+
+| admin_state_up              | True                                 |
+| connection_limit            | -1                                   |
+| created_at                  | 2023-01-29T16:12:24                  |
+| default_pool_id             | None                                 |
+| default_tls_container_ref   | None                                 |
+| description                 |                                      |
+| id                          | c1bba40d-3b45-4b17-86fd-81d98909822e |
+| insert_headers              | None                                 |
+| l7policies                  |                                      |
+| loadbalancers               | eaf6d4f3-8d73-4b77-8bc4-788a3b4e3916 |
+| name                        | mylb-listener-metrics                |
+| operating_status            | OFFLINE                              |
+| project_id                  | dfc700467396428bacba4376e72cc3e9     |
+| protocol                    | PROMETHEUS                           |
+| protocol_port               | 8088                                 |
+| provisioning_status         | PENDING_CREATE                       |
+| sni_container_refs          | []                                   |
+| timeout_client_data         | 50000                                |
+| timeout_member_connect      | 5000                                 |
+| timeout_member_data         | 50000                                |
+| timeout_tcp_inspect         | 0                                    |
+| updated_at                  | None                                 |
+| client_ca_tls_container_ref | None                                 |
+| client_authentication       | NONE                                 |
+| client_crl_container_ref    | None                                 |
+| allowed_cidrs               | ['203.0.113.132/32']                 |
+| tls_ciphers                 | None                                 |
+| tls_versions                | None                                 |
+| alpn_protocols              | None                                 |
+| tags                        |                                      |
++-----------------------------+--------------------------------------+
+```
+
+To check the provisioning status of `mylb-listener-metrics`, type:
+
+```console
+$ openstack loadbalancer listener show mylb-listener-metrics \
+  -c provisioning_status
++---------------------+--------+
+| Field               | Value  |
++---------------------+--------+
+| provisioning_status | ACTIVE |
++---------------------+--------+
+```
+
+## Testing the listener
+
+Once the listener reports as `ACTIVE`, you should be able to check its endpoint.
```

Comment:
```suggestion
Once the listener reports as being `ACTIVE`, you should be able to check its endpoint.
```

(the above reads better --- at least for me)

Reply:
Fair point; fixed as suggested.